backend/dataall/core/environment/api/types.py

```diff
@@ -177,5 +189,15 @@
         gql.Field(name='hasNext', type=gql.Boolean),
         gql.Field(name='hasPrevious', type=gql.Boolean),
         gql.Field(name='nodes', type=gql.ArrayType(ConsumptionRole)),
+        gql.Field(
```

Comment:
I think we do not need these fields, they are already part of `ConsumptionRole` which is the type for each of the `nodes`